assets/svg/pagespeed-insights.svg

```diff
@@ -0,0 +1,39 @@
+                        <path id="_Path_2" data-name="&lt;Path&gt;" d="M12.58 0h158.81a12.53 12.53 0 0 1 12.54 12.54v12.54H0V12.54A12.53 12.53 0 0 1 12.58 0z" fill="#c2c2c2"/>
+                        <path id="_Path_3" data-name="&lt;Path&gt;" d="M171.39 0H12.58A12.57 12.57 0 0 0 0 12.54v1A12.58 12.58 0 0 1 12.58 1h158.81a12.59 12.59 0 0 1 12.54 12.54v-1A12.58 12.58 0 0 0 171.39 0z" fill-opacity=".2" fill="#fff"/>
+                        <path id="_Path_4" data-name="&lt;Path&gt;" class="cls-4" d="M0 25.08h183.93v1H0z"/>
+                        <path id="_Path_5" data-name="&lt;Path&gt;" d="M91.91 62.69a66.76 66.76 0 0 0-66.76 66.86c0 1.4-.09 3.14 0 4.18h29.13a36.26 36.26 0 0 1 0-4.18 37.64 37.64 0 0 1 57.36-32l21-21a66.62 66.62 0 0 0-40.73-13.86z" fill="#4285f4"/>
+                        <path id="_Path_6" data-name="&lt;Path&gt;" d="M132.64 76.52l-21 21a37.59 37.59 0 0 1 17.9 32 36.26 36.26 0 0 1 0 4.18h29.27c.08-1 0-2.78 0-4.18a66.73 66.73 0 0 0-26.17-53z" fill="#f44336"/>
+                        <circle id="_Path_7" data-name="&lt;Path&gt;" class="cls-7" cx="16.76" cy="12.54" r="4.18"/>
+                        <circle id="_Path_8" data-name="&lt;Path&gt;" class="cls-7" cx="33.48" cy="12.54" r="4.18"/>
+                        <path id="_Path_9" data-name="&lt;Path&gt;" class="cls-4" d="M171.41 157.76H12.54A12.57 12.57 0 0 1 0 145.23v1a12.57 12.57 0 0 0 12.54 12.54h158.87a12.58 12.58 0 0 0 12.59-12.5v-1a12.58 12.58 0 0 1-12.54 12.53z"/>
+                        <path id="_Path_10" data-name="&lt;Path&gt;" d="M132.86 79.06a2.06 2.06 0 0 0 .39-1.22 2.1 2.1 0 0 0-2.09-2.09 2 2 0 0 0-1.28.46l-45.57 35.07a12.53 12.53 0 1 0 18.37 16.41l30.17-48.65z" fill="#9e9e9e"/>
+                        <path id="_Path_11" data-name="&lt;Path&gt;" d="M84.31 112.33l45.57-35.09a2.06 2.06 0 0 1 1.28-.46 2.09 2.09 0 0 1 2 1.55 2 2 0 0 0-2-2.59 2 2 0 0 0-1.28.46l-45.57 35.07a12.5 12.5 0 0 0-4.87 9.92v.56a12.49 12.49 0 0 1 4.87-9.42z" style="isolation:isolate" opacity=".2" fill="#fff"/>
+                        <path id="_Path_12" data-name="&lt;Path&gt;" d="M132.86 79.06l-30.17 48.65a12.51 12.51 0 0 1-23.21-5.93v.48a12.52 12.52 0 0 0 23.24 6.5l30.14-48.66a2 2 0 0 0 .39-1.22 1.91 1.91 0 0 0-.08-.54 2.31 2.31 0 0 1-.31.72z" style="isolation:isolate" opacity=".2" fill="#212121"/>
+                        <path id="_Path_13" data-name="&lt;Path&gt;" d="M132.82 76.58a2 2 0 0 1 .43 1.26 2.06 2.06 0 0 1-.39 1.22l-30.17 48.65a12.53 12.53 0 0 1-18.84 3.05l28.08 28.07h59.47a12.53 12.53 0 0 0 12.54-12.54v-18.61z" fill="url(#linear-gradient)"/>
+                    </g>
+                    <path id="_Path_14" data-name="&lt;Path&gt;" d="M183.93 12.54A12.53 12.53 0 0 0 171.39 0H12.58A12.53 12.53 0 0 0 0 12.54v133.73a12.53 12.53 0 0 0 12.54 12.54h158.85a12.53 12.53 0 0 0 12.54-12.54z" fill="url(#radial-gradient)"/>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
I cleaned this up a bit manually – can you check and see that it looks the same?
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 183.95 158.81">
  <defs>
    <radialGradient id="radial-gradient" cx="1360.51" cy="10958.05" r="458.83" gradientTransform="matrix(.5 0 0 .5 -675.69 -5476.24)" gradientUnits="userSpaceOnUse">
      <stop offset="0" stop-color="#fff" stop-opacity=".1"/>
      <stop offset="1" stop-color="#fff" stop-opacity="0"/>
    </radialGradient>
    <linearGradient id="linear-gradient" x1="1572.36" y1="11164.11" x2="1691.74" y2="11283.49" gradientTransform="matrix(.5 0 0 .5 -675.69 -5476.24)" gradientUnits="userSpaceOnUse">
      <stop offset="0" stop-color="#212121" stop-opacity=".2"/>
      <stop offset="1" stop-color="#212121" stop-opacity="0"/>
    </linearGradient>
  </defs>
  <path d="M183.93 25.08v121.19a12.53 12.53 0 01-12.54 12.54H12.58A12.53 12.53 0 010 146.27V25.08z" fill="#e1e1e1"/>
  <path d="M12.58 0h158.81a12.53 12.53 0 0112.54 12.54v12.54H0V12.54A12.53 12.53 0 0112.58 0z" fill="#c2c2c2"/>
  <path d="M171.39 0H12.58A12.57 12.57 0 000 12.54v1A12.58 12.58 0 0112.58 1h158.81a12.59 12.59 0 0112.54 12.54v-1A12.58 12.58 0 00171.39 0z" fill-opacity=".2" fill="#fff"/>
  <path fill="#212121" fill-opacity=".1" d="M0 25.08h183.93v1H0z"/>
  <path d="M91.91 62.69a66.76 66.76 0 00-66.76 66.86c0 1.4-.09 3.14 0 4.18h29.13a36.26 36.26 0 010-4.18 37.64 37.64 0 0157.36-32l21-21a66.62 66.62 0 00-40.73-13.86z" fill="#4285f4"/>
  <path d="M132.64 76.52l-21 21a37.59 37.59 0 0117.9 32 36.26 36.26 0 010 4.18h29.27c.08-1 0-2.78 0-4.18a66.73 66.73 0 00-26.17-53z" fill="#f44336"/>
  <circle fill="#eee" cx="16.76" cy="12.54" r="4.18"/>
  <circle fill="#eee" cx="33.48" cy="12.54" r="4.18"/>
  <path fill="#212121" fill-opacity=".1" d="M171.41 157.76H12.54A12.57 12.57 0 010 145.23v1a12.57 12.57 0 0012.54 12.54h158.87a12.58 12.58 0 0012.59-12.5v-1a12.58 12.58 0 01-12.54 12.53z"/>
  <path d="M132.86 79.06a2.06 2.06 0 00.39-1.22 2.1 2.1 0 00-2.09-2.09 2 2 0 00-1.28.46l-45.57 35.07a12.53 12.53 0 1018.37 16.41l30.17-48.65z" fill="#9e9e9e"/>
  <path d="M84.31 112.33l45.57-35.09a2.06 2.06 0 011.28-.46 2.09 2.09 0 012 1.55 2 2 0 00-2-2.59 2 2 0 00-1.28.46l-45.57 35.07a12.5 12.5 0 00-4.87 9.92v.56a12.49 12.49 0 014.87-9.42z" style="isolation:isolate" opacity=".2" fill="#fff"/>
  <path d="M132.86 79.06l-30.17 48.65a12.51 12.51 0 01-23.21-5.93v.48a12.52 12.52 0 0023.24 6.5l30.14-48.66a2 2 0 00.39-1.22 1.91 1.91 0 00-.08-.54 2.31 2.31 0 01-.31.72z" style="isolation:isolate" opacity=".2" fill="#212121"/>
  <path d="M132.82 76.58a2 2 0 01.43 1.26 2.06 2.06 0 01-.39 1.22l-30.17 48.65a12.53 12.53 0 01-18.84 3.05l28.08 28.07h59.47a12.53 12.53 0 0012.54-12.54v-18.61z" fill="url(#linear-gradient)"/>
  <path d="M183.93 12.54A12.53 12.53 0 00171.39 0H12.58A12.53 12.53 0 000 12.54v133.73a12.53 12.53 0 0012.54 12.54h158.85a12.53 12.53 0 0012.54-12.54z" fill="url(#radial-gradient)"/>
</svg>
```